lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -133,7 +140,7 @@ defmodule LivebookWeb.SessionLive.CellComponent do
     <div class="flex">
       <div class="w-1 rounded-lg relative -left-3" data-element="cell-focus-indicator">
       </div>
-      <div class="w-full">
+      <div id="<%= @cell_view.id %>" class="w-full">
```

Comment:
Instead of setting the id, let's set `data-element="cell-body"` and then on the client you can do:

```js
const cellContent = getCellById(hook.state.focusedCellId).querySelector(`[data-element="cell-body"]`);
```

(`getCellById` is already there)

Reply:
Actually, I am not using this value. So we should actually remove this.  
I get the cell id value from somewhere else.

Reply:
I thought the point is to capture only the cell source + output, and omit the action icons?

Reply:
I see what you mean now @jonatanklosko. This solution is way better. 
Updates coming.